### PR TITLE
chore: migrate to themes v2

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: src/ToDo.UI/App.xaml [hunk 1]
```diff
@@ -13,19 +13,32 @@
...
+
+				<!-- Load Material Control Resources -->
+				<MaterialResourcesV2 xmlns="using:Uno.Material" WithImplicitStyles="True" />
 
-				<!-- Load Material Toolkit resources -->
-				<MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
+				<!-- Load Material Toolkit Resources -->
+				<MaterialToolkitResourcesV2 xmlns="using:Uno.Toolkit.UI.Material" WithImplicitStyles="True" />
 
+				<ResourceDictionary>
```
Comment (agneszitte-nventive) [inline src/ToDo.UI/App.xaml:31]: @kazo0 / @NVLudwig  Only specific to this app for the OverlayColor (for the FlyoutLightDismissOverlayBackground) ? or can be added into themes ?
```
<ResourceDictionary.ThemeDictionaries>
						<ResourceDictionary x:Key="Light">
							<Color x:Key="OverlayColor">#51000000</Color>
						</ResourceDictionary>
						<ResourceDictionary x:Key="Dark">
							<Color x:Key="OverlayColor">#51FFFFFF</Color>
						</ResourceDictionary>
					</ResourceDictionary.ThemeDictionaries>
```

File: src/ToDo.UI/Views/SettingsPage.xaml.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ private void PageLoaded()
 
 		isInitializing = true;
 
-		// Set default theme
+		//// Set default theme
```
Comment (agneszitte-nventive) [inline src/ToDo.UI/Views/SettingsPage.xaml.cs:41]: this change can be removed

File: src/ToDo.UI/App.xaml (hunk 1 above)
Comment (NVLudwig) [reply src/ToDo.UI/App.xaml:31]: This looks like an overlay used for Flyout and such...If so then yes you can keep it for all apps

Comment (agneszitte-nventive) [reply src/ToDo.UI/App.xaml:31]: @kazo0 I think we can move the resources to Uno.Themes then

Comment (kazo0) [reply src/ToDo.UI/App.xaml:31]: I can follow this up with a PR for the overlay color

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**